charts/localstack/templates/deployment.yaml

```diff
@@ -157,6 +157,9 @@ spec:
           volumeMounts:
             - name: dind-storage
               mountPath: /var/lib/docker
+              {{- if ne (len .Values.persistence.extraVolumeMounts) 0 }}
+{{ toYaml .Values.persistence.extraVolumeMounts | indent 12 }}
+          {{- end }}
```

Comment:
This section is within the conditional `if .Values.mountDind.enabled`, which would mean that it would only work if `mountDind.enabled` is `true`.